src/pyodide/internal/_workers.py

```diff
@@ -930,6 +931,50 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class _WorkflowStepWrapper:
+    def __init__(self, js_step):
+        self._js_step = js_step
+
+    def do(self, name, callback=None, config=None):
+        if callback:
+
+            async def wrapper():
+                return _do_call(self, name, config, callback)
+
+            return wrapper()
+
+        # Return a decorator that will execute the function when called
+        def decorator(func):
+            @functools.wraps(func)
+            async def wrapper(*args, **kwargs):
```

Comment:
We seem to be ignoring the arguments. What are the arguments we would be getting here? I think maybe this should also be:
```suggestion
            async def wrapper():
```
Or would that not work for some reason?

Reply:
Yeah we can omit the args here